include/lsst/afw/table/Schema.h

```diff
@@ -100,6 +100,19 @@ class Schema {
     SchemaItem<T> find(Key<T> const & key) const;
 
     /**
+     *  @brief Find a SchemaItem by name and run a functor on it.
+     *
+     *  Names corresponding to named subfields are not accepted.
+     *  The given functor must have an overloaded function call
+     *  operator that accepts any SchemaItem type (the same as
+     *  a functor provided to forEach).
+     */
+    template <typename F>
+    void find(std::string const & name, F && func) const {
```

Comment:
I don't like the name of this function. `find` does not imply "run a functor on it".

`applyTo` / `findAndApply` / `transformItem`? I don't know, but not `find`.